t/include_conf/include_conf.t.in

```diff
@@ -0,0 +1,32 @@
+#!@PERL@
+use strict;
+use Test::More tests => 3; # pack here your amount of subtests
+use SysWrap;
+
+#
+# Write here in brief, what you want to check!
```

Comment:
Um what? Your supposed to replace that with your own doc describing your test...